### ISSUE TITLE
Error after server restart

### ISSUE THREAD

**Issue by Theepwner** [BanManagement/BanManager#395]
http://hastebin.com/arujujenul.avrasm

Build 179

**Comment by Kakifrucht** [comment]
Does this happen every time or only once? Did it generate followup issues?

**Comment by confuser** [comment]
Duplicate of #321

**Comment by EvilOlaf** [comment]
Related #321?
